### DISCUSSION TITLE
Welcome to MSNoise Discussions!

### DISCUSSION THREAD

**Discussion by ThomasLecocq** [ROBelgium/MSNoise#240] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

With this tool, I have the objective to group all MSNoise-related matters here on GitHub, where discussions, bugs/issues, pull requests etc, can be easily linked. If this is proven successful I plan to stop the mailing list server and only use this platform.

Best regards,
Thomas

**Comment by echavess** [comment]
Sounds fantastic! Definitely will be joining.

Cheers

Dr. Esteban J. Chaves
https://echavess.github.io
